File: 백준/Bronze/9498. 시험 성적/시험 성적.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString();

const score = parseInt(input);

score >= 90
  ? console.log("A")
  : score >= 80
  ? console.log("B")
  : score >= 70
  ? console.log("C")
  : score >= 60
  ? console.log("D")
  : console.log("F");

File: 백준/Bronze/9498. 시험 성적/시험 성적.swift
let score = Int(readLine()!)!

if (90...100).contains(score) {
  print("A")
} else if (80...89).contains(score) {
  print("B")
} else if (70...79).contains(score) {
  print("C")
} else if (60...69).contains(score) {
  print("D")
}  else {
  print("F")
}